python/llm/example/GPU/HF-Transformers-AutoModels/README.md

```diff
@@ -0,0 +1,7 @@
+# BigDL-LLM Optimization for Large Language Model on Intel GPUs
+
+To help you quickly get started with transformers-style API to run any Huggingface Transformer model on Intel GPU, we provide lots of detailed examples demonstrating various usage, including:
+
+- [Model Folder](Model): examples to run Huggingface Transformer models with INT4 optimizations.
+- [More-Data-Types Folder](More-Data-Types): examples of how to apply other low bit optimizations.
+- [Save-Load Folder](Save-Load): examples about how to save and load optimized model.
```

Comment:
This folder contains examples of running any Huggingface Transformer model on BigDL-LLM (using the standard AutoModel APIs)
- [Model](Model): examples of running Huggingface Transformer models (e.g., LLaMA2, ChatGLM2, Falcon, MPT, Baichuan2, etc.) using INT4 optimizations
- [More-Data-Types](More-Data-Types): examples of applying other low bit optimizations (NF4/INT5/INT8, etc.)
- [Save-Load](Save-Load): examples of saving and loading low-bit models